pkg/utils/version.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"context"
+
+	semverlib "github.com/Masterminds/semver/v3"
+	"github.com/google/go-github/v45/github"
+)
+
+func CheckLatestVersion(currentVersion string) {
```

Comment:
There is a check for latest version in the `plural version` command already, see `cmd/plural/version.go`.

Reply:
<img width="656" alt="Zrzut ekranu 2022-11-3 o 14 57 52" src="https://user-images.githubusercontent.com/2823399/199740270-e26a83fb-0584-489e-b569-575181e753fa.png">

Reply:
I've added this check as awrapper for all CLI commands